xen/xenops_interface.ml

```diff
@@ -433,6 +433,8 @@ module Vif = struct
 
 	type id = string * string
 
+	type static_ip_setting = (string * string) list
```

Comment:
We could have a type corresponding to the proposed datamodel changes:
```
type ip_configuration =
    | Unspecified
    | Static of string list * string option    (* a list of CIDRs and optionally a gateway *)
```
Then we won't need separate set and unset functions; a single `set_ip_configuration` would do.